### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by barieom**
remove broken link - another update to this page coming soon

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

**Comment by barieom** [comment]
docs team - could you merge this once approved?

**Review by cswatt — APPROVED** [review]

**Pull request merged**
